src/Atmos/Dycore/test/isentropic_vortex.jl

```diff
@@ -19,6 +19,8 @@ using CLIMA.PlanetParameters: R_d, cp_d, grav, cv_d
 
 const halfperiod = 5
 
+const print_diagnostics = length(ARGS) == 0 || parse(Bool, ARGS[1])
```

Comment:
we should use Julia's logging infrastructure for this `@debug`, `@info` for this: want me to have a go?

Reply:
Sure. I was just trying to suppress the existing output in the tests

Reply:
We need it to play well with MPI too (so best would be also some `@root_info` as well in which case only 1 ranks does the output not everyone)

Reply:
You can see how I used this down on line 127 to define an `io` variable.

Reply:
I think we should just be able to set a different log level on each rank.

Reply:
really this is related to larger issue #134

Reply:
theoretically you could set up an MPILogger or something like that which would handle everything automatically, but this will work for now.

Reply:
Sounds good. Are you adding it to this pull request or will you do a separate one?

Reply:
I can just add it to this one.

Reply:
ok

Reply:
Let me know what you think of ce11cbdd510f368cc6733f465c3d2d8e39226917: we could move some of this into Utilities proper.

Reply:
Okay, I've updated all of them, hopefully that should work nicely.